src/debug/ee/debugger.cpp

```diff
@@ -14247,7 +14247,7 @@ Debugger::InsertToMethodInfoList( DebuggerMethodInfo *dmi )
         return (hr);
     }
 
-    DebuggerMethodInfo *dmiPrev = m_pMethodInfos->GetMethodInfo(dmi->m_module, dmi->m_token);
+    DebuggerMethodInfo *dmiPrev = m_pMethodInfos ? m_pMethodInfos->GetMethodInfo(dmi->m_module, dmi->m_token) : NULL;
```

Comment:
CheckInitMethodInfoTable above ensures that `m_pMethodInfos` is non-null. The null check is not necessary here.